fe/fe-core/src/main/java/com/starrocks/catalog/TableProperty.java

```diff
@@ -706,7 +706,8 @@ public TableProperty buildPrimaryIndexCacheExpireSec() {
     }
 
     public TableProperty buildPersistentIndexType() {
-        String type = properties.getOrDefault(PropertyAnalyzer.PROPERTIES_PERSISTENT_INDEX_TYPE, "LOCAL");
+        String defaultType = Config.enable_cloud_native_persistent_index_by_default ? "CLOUD_NATIVE" : "LOCAL";
```

Comment:
better to define static final string CLOUD_NATIVE, LOCAL

Reply:
done